<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButtonPV">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Put Voxel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCV">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cut Voxel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPB">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Put Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCB">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>85</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cut Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPS">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Put Sphere</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCS">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cut Sphere</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPE">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Put Ellipsoid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCE">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>84</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cut Ellipsoid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSAVE">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>61</height>
         </size>
        </property>
        <property name="text">
         <string>Save Project</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>B</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSlider" name="horizontalSliderR">
            <property name="minimumSize">
             <size>
              <width>11</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>16</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderG">
            <property name="minimumSize">
             <size>
              <width>11</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>16</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSliderB">
            <property name="minimumSize">
             <size>
              <width>11</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>16</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="verticalSliderA">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>61</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>61</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_19">
      <item>
       <widget class="Interface" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>21</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>21</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Box</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Size X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderSizeX">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberSizeX">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Size Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderSizeY">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberSizeY">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Size Z</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderSizeZ">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberSizeZ">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Sphere</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderRadius">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberRadius">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>49</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>49</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string>Ellipsoid</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>X Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderXradius">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberXradius">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Y Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderYradius">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberYradius">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="minimumSize">
               <size>
                <width>49</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>49</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Z Radius</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSliderZradius">
              <property name="minimumSize">
               <size>
                <width>151</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>151</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumberZradius">
            <property name="maximumSize">
             <size>
              <width>51</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="horizontalSliderPLANE">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberPLANE">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChange_Size"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChange_Size">
   <property name="text">
    <string>Change Size</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Interface</class>
   <extends>QWidget</extends>
   <header>interface.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
